page/ts/faqScroll.js

```diff
@@ -0,0 +1,73 @@
+/**
+ *
+ *  jQuery plugin that handling the smooth 
+ *  scrolling for FAQ page
+ */
+
```

Comment:
please leave a note that this should be written in TS but due to the nature of this plugin it would probably result with `:any` everywhere and thats why it's just JS.